tests/plugins/widget/integration/table/manual/widgetintable.md

```diff
@@ -0,0 +1,17 @@
+@bender-tags: 1027, 4.8.0, widget
+@bender-ui: collapsed
+@bender-ckeditor-plugins: undo,clipboard,basicstyles,toolbar,wysiwygarea,widget,table,resize
+
+----
+
+1. Click into widgets in table
+1. Try to select some text **in table**
+1. Perform some operation on selected text ( change, append new text, apply some style, etc. )
+1. Check behaviour inside inner and outer table depend of testing widget
```

Comment:
This step is a little unclear. I assume it means to test for both text in nested and outer table. Maybe it can be skipped and added to the instructions below test scenario?